build.sbt

```diff
@@ -1,109 +0,0 @@
-organization in ThisBuild := "sample.chirper"
```

Comment:
By the way, at some point Play may switch to not enabling the PlayLayoutPlugin by default, so it will be an opt in thing.